docs/readthedocs/source/doc/LLM/Quickstart/axolotl_quickstart.md

```diff
@@ -0,0 +1,214 @@
+pip install datasets==2.15.0
+# prepare axolotl entrypoints
+wget https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/GPU/LLM-Finetuning/axolotl/finetune.py
+wget https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/GPU/LLM-Finetuning/axolotl/train.py
+```
+
+**After the installation, you should have created a conda environment, named `axolotl` for instance, for running `Axolotl` commands with IPEX-LLM.**
+
+### 2. Example: Finetune Llama-2-7B with Axolotl
+
+The following example will introduce finetuning [Llama-2-7B](https://huggingface.co/meta-llama/Llama-2-7b) with [alpaca_2k_test](https://huggingface.co/datasets/mhenrichsen/alpaca_2k_test) dataset using LoRA and QLoRA.
+
+Note that you don't need to write any code in this example.
+
+| Model | Dataset | Finetune method |
+|-------|-------|-------|
+| Llama-2-7B | alpaca_2k_test | LoRA (Low-Rank Adaptation)  |
+| Llama-2-7B | alpaca_2k_test | QLoRA (Quantized Low-Rank Adaptation) |
+
+For more technical details, please refer to [Llama 2](https://arxiv.org/abs/2307.09288), [LoRA](https://arxiv.org/abs/2307.09288) and [QLoRA](https://arxiv.org/abs/2305.14314).
+
+#### 2.1 Download Llama-2-7B and alpaca_2k_test
+
+By default, Axolotl will automatically download models and datasets from Huggingface. Please ensure you have login to Huggingface.
+
+```cmd
+huggingface-cli login
+```
+
+If you prefer offline models and datasets, please download [Llama-2-7B](https://huggingface.co/meta-llama/Llama-2-7b) and [alpaca_2k_test](https://huggingface.co/datasets/mhenrichsen/alpaca_2k_test). Then, set `HF_HUB_OFFLINE=1` to avoid connecting to Huggingface.
+
+```cmd
+export HF_HUB_OFFLINE=1
+```
+
+#### 2.2 Set Environment Variables
+
+```eval_rst
+.. note::
+
+   This is a required step on for APT or offline installed oneAPI. Skip this step for PIP-installed oneAPI.
+```
+
+Configure oneAPI variables by running the following command:
+
+```eval_rst
+.. tabs::
+   .. tab:: Linux
+
+      .. code-block:: bash
+
+         source /opt/intel/oneapi/setvars.sh
+
+```
+
+Configure accelerate to avoid training with CPU
+
+```cmd
+accelerate config
+```
+
+Please answer `NO` in option `Do you want to run your training on CPU only (even if a GPU / Apple Silicon device is available)? [yes/NO]:`.
+
+After finishing accelerate config, check if `use_cpu` is disabled (i.e., `use_cpu: false`) in accelerate config file (`~/.cache/huggingface/accelerate/default_config.yaml`).
+
+#### 2.3 LoRA finetune
+
+Prepare `lora.yml` for Axolotl LoRA finetune. You can download a template from github.
+
+```cmd
+wget https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/GPU/LLM-Finetuning/axolotl/lora.yml
+```
+
+If you use the model and dataset in local env, please modify the model path and dataset path in `lora.yml` to ensure Axolotl can access the offline model and data.
+
+```yaml
+# Please change to local path if model is offline
+base_model: /path/to/model/Llama-2-7b-hf
```

Comment:
should we just use model/dataset id in hf by default?

Reply:
Yes. Change back to default and add comments